static/js/topic_page/map_tile.tsx

```diff
@@ -176,15 +177,23 @@ function fetchData(
     populationPromise,
     parentPlacesPromise,
   ])
-    .then(([geoJson, placeStat, population, parentPlaces]) => {
+    .then(([geoJson, placeStat, populationData, parentPlaces]) => {
+      const metadataMap = placeStat.facets;
+      if (!_.isEmpty(populationData.facets)) {
+        Object.assign(metadataMap, populationData.facets);
+      }
+      let population = {};
+      if (
+        !_.isEmpty(populationData.data) &&
+        denomStatVar in populationData.data
+      ) {
+        population = populationData.data[denomStatVar];
+      }
```

Comment:
perhaps, this should check if denomStatVar is defined in the first place. checking for `"" in object` works but is error prone.

also see comment above about always using a denominator for this..

Reply:
Done!